mpf/devices/multiball_lock.py

```diff
@@ -305,3 +305,26 @@ def _post_events(self, device, **kwargs):
     def _request_new_balls(self, balls):
         """Request new ball to playfield."""
         self.source_playfield.add_ball(balls=balls)
+    
+    def compensate_lock(self):
+        # evilmopey solution for issue 1278
+        if self.config['locked_ball_counting_strategy'] == "no_virtual":
```

Comment:
If this approach is only applicable to the `"no_virtual"` approach, could you wrap the calling function above instead of having so many conditionals?

```
if self.config['locked_ball_counting_strategy'] == "no_virtual":
  self.compensate_lock()
```